builder/common/ami_config_test.go

```diff
@@ -283,3 +283,78 @@ func TestAMINameValidation(t *testing.T) {
 	}
 
 }
+
+// Make sure only valid ARNs are accepted
+func TestAMIARNValidation(t *testing.T) {
+	tests := []struct {
+		name      string
+		OrgARN    []string
+		OUARN     []string
+		expectErr bool
```

Comment:
I would consider extending this to check the specific error returned, since there's several different cases that could change and these tests wouldn't fail

Reply:
I'm less sure about this, I think checking if the configs provoke an error when expected is more important than the contents of those errors, so I'm tempted to leave it as-is, but please feel free to push back if you think it's important